swift/read_file.swift

```diff
@@ -0,0 +1,19 @@
+//Read a file (file path given as a command line argument) and write to stdout
+
+
+import Foundation
+
+guard CommandLine.arguments.count > 1 else {
+    print("Usage: swift script.swift <file_path>")
+    exit(1)
+}
+
+let fileURL = URL(fileURLWithPath: CommandLine.arguments[1])
+let fileContents = try String(contentsOf: fileURL)
```

Comment:
```suggestion
let fileContents = try String(contentsOfFile: CommandLine.arguments[1])
```

If we use `contentsOfFile`, we don't need to make `URL` from the command line argument, we can pass it straight to `String(...)`